flink-kubernetes/src/main/java/org/apache/flink/kubernetes/KubernetesResourceManagerDriver.java

```diff
@@ -65,6 +67,11 @@
 import java.util.concurrent.CancellationException;
 import java.util.concurrent.CompletableFuture;
 
+import static io.fabric8.kubernetes.client.Watcher.Action.ADDED;
```

Comment:
It's better not use fabric classes in KubernetesResourceManagerDriver since these are Flink internal logic.

Reply:
Thanks for your suggestion. It is indeed better not to introduce an external class, I will use an internal enum to replace it.